docs/1.intro.md

```diff
@@ -35,11 +36,22 @@ It acts as a central registry, making it easier for different parts of WordPress
 ## Registration Example
 
 ```php
+// First, register a category
+add_action( 'abilities_api_category_registry_init', 'my_plugin_register_category');
```

Comment:
Nitpick, it could help to find a shorter name for the action
```suggestion
add_action( 'abilities_category_init', 'my_plugin_register_category');
```

or a bit differently structured than `abilities_api_init` but more in natural English:

```suggestion
add_action( 'ability_categories_init', 'my_plugin_register_category');
```